egs2/sdsv21/spk1/local/sampledeepmine_prep.py

```diff
@@ -0,0 +1,67 @@
+# sampledeepmine_prep.py
+
+# This script makes wav.scp utt2spk and spk2utt files from the DeepMine sample subset
+
+import argparse
+import os
+
+
+def generate_output(source_dir, output_dir, input_file):
+    spk2utt = {}
+    utt2spk = []
+    wav_list = []
+
+    with open(input_file, "r") as f:
+        paths = f.read().splitlines()
```

Comment:
is it different to `paths = f.readlines()`?

Reply:
Yeah, slightly. It removes the newline.